hosts/hetzner-ephemeral-beta/default.nix

```diff
@@ -16,6 +16,9 @@
   # Use stable kernel
   boot.kernelPackages = pkgs.linuxPackagesFor (pkgs.linux);
 
+  # Saiko's automatic gc
+  sys2x.gc.useDiskAware = true;
```

Comment:
How does this work? I guess it's imported by nixobolus?

Reply:
Thanks, this should not be in here. I guess that something went wrong when merging from `main`. Flake check does not errors at the moment because this configuration fail anyways at the moment.

It is a part of pull request #39, commit [d4bca17](https://github.com/ponkila/homestaking-infra/pull/39/commits/d4bca176c4d42e2a450b58349846dccd90b2d236) to be precise.